components/elm/tools/interpinic/src/interpinic.F90

```diff
@@ -146,9 +146,22 @@ subroutine interp_filei (fin, fout, cmdline)
     call check_ret (nf90_open(fin,  NF90_NOWRITE, ncidi ))
     call check_ret (nf90_open(fout, NF90_NOWRITE, ncido ))
     call check_ret (nf_inq_format( ncido, ncformat ))
-    if ( ncformat /= NF_FORMAT_64BIT )then
-       write (6,*) 'error: output file is NOT in NetCDF large-file format!'
-       stop
+    ! 20240822 csz++
+    ! Allow any format for output dataset
+    ! if ( ncformat /= NF_FORMAT_64BIT )then
+    ! write (6,*) 'error: output file is NOT in NetCDF large-file format!'
+    ! stop
+    ! 20240822 csz--
```

Comment:
Just remove the commented out code.  Here and below.

Reply:
Done. Also fixed a problem exposed by new compilers that required removing `external` from `iargc` definition that was already in master but not in Gautam's `initerpinic` branch that I started from. Verified that new code compiles without `USER_FFLAGS=-fallow-invalid-boz` thanks to Xylar's patch.